src/field.rs

```diff
@@ -37,6 +37,6 @@ macro_rules! field {
     ($struct_name:ident, $field_name:ident) => {{
         let opt = None::<&$struct_name>.map(|v| &v.$field_name);
         let offs = $crate::__glium_offset_of!($struct_name, $field_name);
-        $crate::field::_hidden_field(offs, size, opt)
```

Comment:
Could you maybe remove the size param entirely? It's not needed for anything.

Reply:
I've went ahead and removed the size parameter :)